docs/1.guide/5.serverless.md

```
@@ -0,0 +1,29 @@
+---
+icon: clarity:cloud-traffic-line
+---
+
+# Serverless
+
+> Deploy to any serverless platform
+
+`srvx` provides built-in support for serverless platforms, allowing you to deploy your applications to various serverless environments without changing your code.
+
+### AWS Lambda
```

markdownlint-cli2 (0.20.0)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)